bolts/data/datamodules/vision/__init__.py

```diff
@@ -0,0 +1 @@
+from .mnist import *
```

Comment:
I can't comment on the file below. But it looks like our mnist version has gone :(

Reply:
The dataset is still present; I only deleted the dm code, because, as I said above, it needs to be reimplemented from scratch, something which is beyond the scope of this PR.